src/main/java/com.gigssandbox/MainClass.java

```diff
@@ -1,10 +1,10 @@
 package com.gigssandbox;
 
+import com.gigssandbox.IO.console.ConsoleInputHandler;
+import com.gigssandbox.IO.console.ConsoleOutputHandler;
 
 public class MainClass {
     public static void main(String[] args) {
-        UserInteraction interaction = new UserInteraction();
-
-        interaction.startInteraction();
+        new UserActivityManager(new ConsoleInputHandler(), new ConsoleOutputHandler()).startInteracting();
```

Comment:
but right now you have only one implementation. you can use DI later. Always try to avoid write code which is aimed for the future because such code bothers you and imposes restrictions the way you thinking. Code should solve current problem not future one